src/Microsoft.DotNet.Build.Tasks/PackageFiles/MSBuild.runtimeconfig.json

```diff
@@ -2,7 +2,7 @@
   "runtimeOptions": {
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "1.0.0-rc3-002733"
```

Comment:
We shouldn't do this now since as you say it makes it unmergable. There are bigger changes that will have to happen in order for this to work as expected, for instance, we need to:
-Update all of the runtimeconfig.json in the buildtools package since now all of them will run with RTM shared framework. (this is the easy part)
- Make all of the tools compile against RTM shared framework since that is what they will have at runtime. (This is the hard part, since it is not only updating the dependencies in buildtools, but also the ones for msbuild and for roslyn)
- Update the CLI in all of the repos that consume buildtools, in order for them to be able to consume this change.

cc: @mellinoe